woocommerce/woocommerce-android
# Unified Login: Jetpack not connected error screens

Closes #2657 by implementing new designs and button options on the "Jetpack not connected" screen:

Light | Dark
-- | --
![Screenshot_1599080606](https://user-images.githubusercontent.com/5810477/92036546-66ff8b00-ed3e-11ea-81ac-22ac29b3fb4e.png)|![Screenshot_1599080612](https://user-images.githubusercontent.com/5810477/92036549-67982180-ed3e-11ea-9528-f15b6b9f92f6.png)

## To Test
1. Tap **Enter your store address**
2. Enter a store address where Jetpack is either not installed or not active/connected.
3. Verify the Jetpack Required screen is displayed (see screenshot above)
4. Verify store address populates in the message.
5. Click **What is Jetpack** and verify the "What is jetpack" dialog appears. Close dialog.
6. Tap **Learn how to install and connect Jetpack** and verify help page loaded successfully
7. Tap to go back to Jetpack required screen.
8. Tap **Log in with another account** - verify routed back to the login prologue screen


<img src="https://user-images.githubusercontent.com/5810477/92036977-0fadea80-ed3f-11ea-955c-cdd0f4307c16.gif" width="330"/>



Update release notes:
